### ISSUE TITLE
Backend Developer / DevOps Engineer - Ruby on Rails

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#3300]
Backend Developer / DevOps Engineer - Ruby on Rails<br>
http://ift.tt/2uaLTN6<br><p><img alt="Logo.gif?ixlib=rails-2.1" src="http://ift.tt/2uat24x"></p>
<p><strong>Headquarters:</strong> Stuttgart, Germany<br><strong>URL:</strong> <a href="http://virtualq.io">http://virtualq.io</a></p>
<div>virtualQ provides a <b>novel cloud-solution</b> that waits on hold during longer waiting times for callers of Customer Service Center. Callers can spend their time productively until virtualQ alerts them to call back when it is their turn. Thereby, the <b>virtualQ enterprise software</b> provides an <b>unique customer experience</b>, which is <b>revolutionising the service branche</b>. Well-known companies like <b>Albatros Lufthansa</b>, the German insurance company <b>HanseMerkur</b> and the international cashback and shopping point provider <b>Lyoness</b> are already using our software.&nbsp;<span>Several national and international institutions awarded the innovative cloud-products of virtualQ and the company is backed by an institutional investment fund.</span>
</div>
<div><br></div>
<div>
<span><span><span>You get the chance to work on an innovative and awarded product with <b>top-notch technology and software management tools</b>. We have a German-American background and like the concept of <b>simplicity, design, flat hierarchy and employee participation</b>. You have the chance to work in an awesome team atmosphere with lots for individual liberties. Do you want to change the way in which people wait on hold in customer service telephone lines with us? Let us know</span></span></span><span>!</span><span><br></span>
</div>
<div><span><br></span></div>
<div><br></div>
<div><b>WITH THESE ACTIVITIES YOU WILL HAVE A GREAT IMPACT ON OUR BUSINESS</b></div>
<div><br></div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <b><strong>Work closely with our CTO</strong></b> <span>and shape our product</span><br>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <b><strong>Design Software and Architecture for our Ruby on Rails Backend</strong></b><span>, using the latest technologies</span><br>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>Design <b>Voice and Audio Applications / Machine Learning Algorithms</b> and <b>Interfaces</b>, that will shape the future of virtualQ</span>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>Interface with APIs like <b>Twilio API, Google Machine Learning, Amazon Alexa Voice Services, IBM Watson</b></span>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <b><strong>Lead a team</strong></b> <span>of great Front and Backend developers</span><br>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>Support <b>DevOps of Amazon AWS hosted Servers, Heroku</b> etc.</span>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>Develop <b>Monitoring, Uptime and Security Strategies</b></span>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <b><strong>Code!!!</strong></b>
</div>
<div><span>&nbsp;</span></div>
<div><span><br></span></div>
<div><b>WE HAVE GOT A LOT TO OFFER</b></div>
<div><br></div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>A <b>fair salary</b> &ndash; we might be a startup, but we pay like a company</span>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <b><strong>Flexibility at its best</strong></b><span>: work remotely from home or join us in Stuttgart or Berlin</span><br>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>Results Only Work Environment &ndash; we do not care when and where you work, we care about your work results. Therefore, we do not count working hours and holidays either</span>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>Plenty of <b>freedom and opportunities to grow</b> in a young company, which will turn the Service Center market upside down</span>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <b><strong>Workshops</strong></b><span>? <b>Seminars</b>? <b>BarCamps</b>? Tell us what you need to develop your skills and we will make it happen</span><br>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>An <b>awesome, inter</b></span><b><strong>national, open minded team<br></strong></b>
</div>
<div><br></div>
<div><br></div>
<div><b>WITH THESE SKILLS YOU ARE OUR PERFECT CANDIDATE&nbsp;</b></div>
<div><br></div>
<div><b><strong>MUST HAVES</strong></b></div>
<div><br></div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>You know <b>Ruby on Rails</b> by heart</span>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>You are familiar with <b>AWS</b></span>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>You speak German OR English fluently (native or 2nd language)</span>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>You have got a <b>technical education / background</b></span>
</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>You do not take things for granted, instead you question the status quo</span>
</div>
<div><span>&nbsp;</span></div>
<div><br></div>
<div><b><strong>NICE TO HAVE</strong></b></div>
<div><br></div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>You have worked with some of the following systems / languages:<span>&nbsp;</span></span>
</div>
<div>
<span>&nbsp; &nbsp; &nbsp; &nbsp; OpsWorks,&nbsp;</span>Chef or Puppet, Containers / Docker, Linux Servers / Web Servers, Open Source Databases, Machine Learning, Algorithms</div>
<div>
<span><span>&#9679;<span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span> <span>You have worked in the Telecommunication market before</span>
</div>
<div><br></div>
<p><strong>To apply:</strong> Please send your CV and you letter of motivation to Anika e-mail: <a href="mailto:%44%72%65%61%6d%6a%6f%62@%76%69%72%74%75%61%6c%51.%69%6f">Dreamjob@virtualQ.io</a></p>
<br><br>
via Remote Jobr Programming Jobs http://ift.tt/1WipnMk<br>
July 20, 2017 at 03:05AM
